core/src/main/java/io/kestra/core/utils/TruthUtils.java

```diff
@@ -0,0 +1,11 @@
+package io.kestra.core.utils;
+
+import java.util.List;
+
+abstract public class TruthUtils {
+    private static final List<String> FALSE_VALUES = List.of("false", "0", "-0", "");
```

Comment:
there is no null possible in pebble?

Reply:
No, it generates an error so I had to do it this way

Reply:
I mean `List.of("false", "0", "-0", "", "null")`, not sure it exists